### PR TITLE
checking generated SLES topics for "p" products

### PR DESCRIPTION
**Pull request by dwicinas**
Products checked are PostGIS, PgBouncer and PgPool II.

Still working on extension topics for PgBouncer

### CODE REVIEW

File: install_template/renders/edb-pgbouncer_1.16_sles-12_ppc64le.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+zypper refresh
+```
+
+## Install the package
+
+```shell
+zypper -n install edb-pgbouncer<xx>
+```
+
+Where `<xx>` is the version of EDB PgBouncer you are installing.
```
[the remark this answers is not in the document]
Comment (dwicinas) [reply install_template/renders/edb-pgbouncer_1.16_sles-12_ppc64le.mdx:45]: @jericson-edb Can you remind me whether we have `{{product.version}}` already defined in the template? I'm not seeing it. If it's already there, can I just plug in `{{product.version}}`?

[the remark this answers is not in the document]
Comment (jericson-edb) [reply install_template/renders/edb-pgbouncer_1.16_sles-12_ppc64le.mdx:45]: `{{product.version}}` is supplied by the template system. It's one of the axes we use to pick which template is used::
* {{product.name}}
* {{product.version}}
* {{platform.name}}
* {{platform.version}}
* {{platform.arch}}

We can always use one of those variables and know they've been defined.

**Comment by josh-heyer** [comment]
@dwicinas I've rebased this onto your rebased epic/templatized-install-guides branch, as there were some oddities in your merge this morning: specifically, that merge looks to have revoked your changes to install_template/templates/platformBase/sles-15.njk in d1a70e340c7e514c6d8c67674d164766cf6eb1dd - if that was intended, let's make it explicit. Otherwise, sanity-check and merge.

**Comment by dwicinas** [comment]
@drothery-edb Similar comment to the connector products PR. I made some changes to the template to reflect the most current approach. Now looks good to me. Let me know if you want to review and commit or should I.

**Pull request merged**